### ISSUE TITLE
Pending listings don't show in Recent listings widget

### ISSUE THREAD

**Issue by bobbie-d** [Strategy11/business-directory-plugin#202]
@stephywells Not sure if this is as intended, but when you have the setting checked for listings to be in pending status before going live, they do not show up in the recent listings widget on the dashboard. It might be helpful to add that, as people may end up checking here for information on listings that need approval.

**Comment by stephywells** [comment]
@bobbie-d yeah, this isn't a bug. Any pending listings will not show on the front-end. All the pending listings have to be approved from the back-end. I'll go ahead and close this. You're welcome to add documentation on this if you'd like.
